main.js

```diff
@@ -0,0 +1,117 @@
+const AppModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+      videos: new VideosCollection(),
+      featuredVideo: null
+    }
+  }
+});
+
+const AppView = Backbone.View.extend({
+  el: $('body'),
+  
+  events: {
+    'click #search-button': 'searchForVideos',
+  },
+  
+  initialize: function () {
+    this.model.get('videos').fetch({reset: true});
```

Comment:
Not the worst thing, but it's a border-line no no for the view to fetch on it's own.